Frame.cc

```diff
@@ -1045,9 +1059,12 @@ bool Frame::GetImageHistogram(int channel, int stokes, int num_bins, CARTA::Hist
         int image_num_bins(current_stats.histogram_bins.size());
 
         if ((num_bins == AUTO_BIN_SIZE) || (num_bins == image_num_bins)) {
+            double min(current_stats.basic_stats[CARTA::StatsType::Min]);
```

Comment:
I am a little uncomfortable with using `min` and `max` as variable names, considering so many other namespaces will do so. All it takes is for us to include a header file that adds `using namespace std` for this code to break. Could we call them `min_val` and `max_val`?

Reply:
Done.